source/clear-linux/get-started/get-started.rst

```diff
@@ -0,0 +1,23 @@
+.. _get-started:
+
+Get started
+###########
+
+This section is where you will find documentation to do with the
+installation of |CL|.
+
+The :ref:`get-started` section provides end to end instructions for
```

Comment:
consider changing "end-to-end" to "step-by-step" and "for downloading and running" to "to download and run"